### PR TITLE
chore: release build-onchain-apps cli

### PR DESCRIPTION
**Pull request by alvaroraminelli**
**What changed? Why?**
- Turn root project private and release the CLI package.

To get started, open a new shell and run:

```
npx @base-org/build-onchain-apps@latest create buy-me-a-coffee-app
```

**Notes to reviewers**

**How has it been tested?**

### CODE REVIEW

File: .github/actions/setup/action.yml [hunk 1]
```diff
@@ -4,13 +4,14 @@ runs:
...
       with:
-        node-version-file: ".nvmrc"
+        node-version-file: '.nvmrc'
         registry-url: https://registry.npmjs.org
         cache: yarn
 
     # For provenance https://docs.npmjs.com/generating-provenance-statements#prerequisites
     - name: Install npm 9.5
       run: npm install -g npm@^9.5.0
+      shell: bash
```
Comment (alvaroraminelli) [inline .github/actions/setup/action.yml:14]: CI is broken missing shell: https://github.com/base-org/build-onchain-apps/actions/runs/6873486350

File: package.json [hunk 2]
```diff
@@ -1,6 +1,7 @@
 {
   "name": "@base-org/build-onchain-apps",
```
Comment (alvaroraminelli) [inline package.json:2]: moving away of standalone repo

File: packages/cli/package.json [hunk 3]
```diff
@@ -1,10 +1,14 @@
 {
   "name": "build-onchain-apps",
```
Comment (alvaroraminelli) [inline packages/cli/package.json:2]: The package name will define how we call the CLI

```bash
npx build-onchain-apps@latest create buy-me-a-coffee-app
```

File: .github/workflows/node.js.yml [hunk 4]
```diff
@@ -16,17 +16,17 @@ jobs:
...
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: 'npm'
+      - name: Install dependencies
+        run: npm install
+      - name: Lint Check
+        # When fails, please run "npm run lint" to your code
+        run: npm run lint
+      # - name: Format Check
```
Comment (alvaroraminelli) [inline .github/workflows/node.js.yml:30]: Format check is breaking the CI. It no longer supported by nx team, see [here](https://github.com/nrwl/nx/issues/4159)

**Review by Zizzamia — APPROVED** [review]
Great update!

**Pull request merged**